src/styles/_header.scss

```
@@ -25,6 +25,11 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: ease-in-out 0.3s;
```

add transform property 👇
```suggestion
    transition: color 0.3s ease-in-out;
```